sam/template.yaml

```diff
@@ -31,14 +31,18 @@ Resources:
   ExasolCloudWatchAdapter:
     Type: AWS::Serverless::Function
     Properties:
-      CodeUri: ../
+      CodeUri: ../target/exasol-cloudwatch-adapter.jar
```

Comment:
I assume these values are default for the template. Can you set another value for parameter when using the SAM cli?